python practice/day17/solution.py

```diff
@@ -3,6 +3,7 @@ class Calculator:
     def power(n,p):
         if p < 0 or n < 0:
             print('n and p should be non-negative')
```

Comment:
This is an extra line.
The problem is said to be raise an exception with a message, just the following line is enough:
raise ValueError('n and p should be non-negative')